zerver/tests/test_subs.py

```diff
@@ -3031,17 +3031,17 @@ def test_previously_subscribed_private_streams(self) -> None:
         self.unsubscribe(admin_user, stream_name)
         self.unsubscribe(non_admin_user, stream_name)
 
-        # Test non admin user shouldn't get previously subscribed private stream's subscribers.
+        # Test admin user shouldn't get previously subscribed private stream's subscribers.
```

Comment:
This is still backwards; it should be "Test admin user gets previously subscribed ...".  Fixed and will merge.